Source/FridgeCache.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Verse;
+using System.Collections.Generic;
+using HarmonyLib;
+
+namespace RimFridge
+{
+    [StaticConstructorOnStartup]
+    public static class FridgeCache
+    {
+        public static Dictionary<IntVec3, CompRefrigerator>[] FridgeGrid = new Dictionary<IntVec3, CompRefrigerator>[10];
```

Comment:
Are we guaranteed to have a maximum of 10 maps?
This really needs to be a List if not a Dictionary

Reply:
Yes the game have a hard limit of 5 maps (colonies), and i don't think many users use more 10 maps at a time.

Reply:
I'm not having a staticly sized array in code. I've seen this go bad too many times in my life.